packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1266,8 +1266,13 @@ export const audiusBackend = ({
     metadata: TrackMetadata,
     onProgress: (loaded: number, total: number) => void
   ) {
-    const storageV2Enabled = await getFeatureEnabled(FeatureFlags.STORAGE_V2)
-    if (storageV2Enabled) {
+    const storageV2SignupEnabled = await getFeatureEnabled(
+      FeatureFlags.STORAGE_V2_SIGNUP
+    )
+    const storageV2UploadEnabled = await getFeatureEnabled(
+      FeatureFlags.STORAGE_V2_TRACK_UPLOAD
+    )
+    if (storageV2SignupEnabled || storageV2UploadEnabled) {
```

Comment:
Why the `or` here?

Reply:
if you signed up with the v2 flow then you won't have a replica set, so you have to upload through the v2 flow. we're gonna rollout upload before signup since once you signup there's no going back (without extra bandaid work, that is)